net-p2p/retroshare/retroshare-0.6.4.recipe

```diff
@@ -0,0 +1,196 @@
+SUMMARY="An open source application that offers secure chat and file sharing"
+DESCRIPTION="Retroshare creates encrypted connections to your friends. Nobody \
+can spy on you. Retroshare is completely decentralized. This means there are \
+no central servers. It is entirely Open-Source and free. There are no costs, \
+no ads and no Terms of Service."
+HOMEPAGE="http://retroshare.sourceforge.net/"
+COPYRIGHT="	1989-1991 Free Software Foundation Inc.
```

Comment:
```diff
-COPYRIGHT="	1989-1991 Free Software Foundation Inc.
+COPYRIGHT="1989-1991 Free Software Foundation Inc.
```

But I think we should move "RetroShare Team" to be the first entry, as they're the one who wrote the program in the first place.